lib/executor.js

```diff
@@ -230,13 +230,19 @@ function runScripts(app, list, callback) {
   var functions = [];
   list.forEach(function(filepath) {
     debug('Requiring script %s', filepath);
-    var exports = require(filepath);
-    if (typeof exports === 'function') {
-      debug('Exported function detected %s', filepath);
-      functions.push({
-        path: filepath,
-        func: exports
-      });
+    try {
+      var exports = require(filepath);
+      if (typeof exports === 'function') {
+        debug('Exported function detected %s', filepath);
+        functions.push({
+          path: filepath,
+          func: exports
+        });
+      }
+    } catch (err) {
+      console.error('Failed loading boot script: %s\n%s',
+        path.basename(filepath), err.message);
```

Comment:
> Why path.basename instead of the full path?

It just made the output easier to read but if you had duplicate script names in different directories it could be a problem.

Your solution looks good, I'll update with that.